src/model_metric_reporter.hpp

```diff
@@ -153,7 +153,7 @@ class ModelMetricReporter : public ServableMetricReporter {
     ModelMetricReporter(const MetricConfig* metricConfig, MetricRegistry* registry, const std::string& modelName, model_version_t modelVersion);
 };
 
-class MediapipeServableMetricReporter {
```

Comment:
you inherited from ServableMetricReporter which introduced plenty of metrics which MediapipeServableMetricReporter does not need

Reply:
does it introduce new lines with value 0 when those are collected? for example TFS metrics